client/src/GamemasterInGame/PlayerListComponents/PlayerMessage/CharacterSelectedYouMessage.tsx

```diff
@@ -0,0 +1,45 @@
+import { PlayerMessageMap, Role } from "@hidden-identity/server";
+import { pluck } from "../../../utils/shuffleList";
+import { RoleSelect } from "../Selectors";
+import { Restrictions } from "./Restrictions";
+import { Flex, Heading } from "@radix-ui/themes";
+import { useState } from "react";
+import { PlayerMessageLink } from "./PlayerMessageLink";
+import { useDefiniteGame } from "../../../store/GameContext";
+
+interface CharacterSelectedYouMessageProps {
+  message: PlayerMessageMap["character-selected-you"];
+}
+
+export function CharacterSelectedYouMessage({
+  message,
+}: CharacterSelectedYouMessageProps) {
+  const { script } = useDefiniteGame();
+  const [role, setRole] = useState<Role>(() =>
+    pluck(script.map(({ id }) => id)),
+  );
+
+  return (
+    <Flex direction="column" gap="2">
+      <PlayerMessageLink
+        className="mb-2"
+        note={{
+          reveal: {
+            "In play": [
```

Comment:
Should this specify: "You've been targeted by"?

Reply:
Nah, I think for most targeted actions it is all the same.